<launch>
  <arg name="joy_dev" default="/dev/input/js0" />

  <node pkg="joy" type="joy_node" name="joy_node">
    <param name="dev" value="$(arg joy_dev)" />
    <param name="deadzone" value="0.3" />
    <param name="autorepeat_rate" value="20" />
  </node>

  <node pkg="printeps_cafe_motion_pepper" name="teleop_twist_joy" type="teleop_twist_joy">
    <param name="linear_speed_max" value="0.10" />
    <param name="angular_speed_max" value="0.15" />
  </node>
</launch>
